common/persistence/visibility/store/elasticsearch/visibility_store.go

```diff
@@ -1462,9 +1463,9 @@ func validateString(value string) error {
 	if len(value) > maxStringLength {
 		return serviceerror.NewInvalidArgument(
 			fmt.Sprintf(
-				"Strings with more than %d bytes are not supported in Elasticsearch (got %s)",
+				"strings with more than %d bytes are not supported in Elasticsearch (got string of len %d)",
 				maxStringLength,
-				value,
+				len(value),
```

Comment:
I don't think that we want to log error message with bytes array which exceeds 32766 bytes.